app/src/main/java/org/simlar/widgets/ContactsAdapter.java

```diff
@@ -42,6 +42,7 @@
 	private final int mLayout;
 	private final LayoutInflater mInflater;
 
+	@SuppressWarnings("ComparatorNotSerializable")
```

Comment:
Letting it implement Serializable and adding the serialVersionUID shouldn't hurt either.